src/app/modules/users/users-list/users-list.component.ts

```diff
@@ -51,7 +54,8 @@ export class UsersListComponent implements OnInit {
   filterType: string = '';
 
   constructor(public dialog: MatDialog, private usersService: UsersService,
-    public cdr: ChangeDetectorRef, private _snackBar: MatSnackBar, ) {
+    public cdr: ChangeDetectorRef, private _snackBar: MatSnackBar, private router: Router,
+    private commonServiceService: CommonServiceService) {
```

Comment:
why are we using this service here?

Reply:
for matsnackbar i wrote one common method in common service